tools/c7n_gcp/c7n_gcp/resources/network.py

```diff
@@ -35,6 +36,25 @@ def get(client, resource_info):
                 'get', {'project': project, 'network': network})
 
 
+@Network.filter_registry.register('vpc-firewall-filter')
```

Comment:
re keeping the user visible name free of redundancy, drop `vpc` and `filter` in the name.

Reply:
fixed